@xen-orchestra/web/src/layouts/AppLayout.vue

```diff
@@ -19,13 +19,14 @@
       <AccountMenu />
     </template>
     <template #sidebar-header>
-      <SidebarSearch v-model="filter" :is-searching />
+      <SidebarSearch v-model="filter" />
     </template>
     <template #sidebar-content>
       <VtsTreeList v-if="!isReady">
         <VtsTreeLoadingItem v-for="i in 5" :key="i" icon="fa:city" />
       </VtsTreeList>
       <NoResults v-else-if="sites.length === 0" />
```

Comment:
```suggestion
      <VtsStateHero v-else-if="sites.length === 0" format="card" size="medium" type="no-result">
        {{ t('no-results') }}
      </VtsStateHero>
```
You can even use this